### PR TITLE
feat: added the acronym task

### PR DESCRIPTION
**Pull request by Captain-Vorobey**

### CODE REVIEW

File: acronym/acronym_test.rb [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+  def self.run(arg_v)
+    for i in 0 ... arg_v.length
+        puts "#{i} #{arg_v[i]}"
+     end
+  end
+end
+
+class Acronym
+    def abbreviate
+        ARGV[0].split("-").join(" ").split.map(&:chr).join.upcase
```
Comment (dzmitry-aleinik-itechart) [inline acronym/acronym_test.rb]: Let's also try to handle the following cases: 
String can contain symbols like &, %, $, — etc. example:
 1. input: 'Ernst&Yong'
     output: 'EY'
 2. input: 'Something — I made up from thin air'
     output: 'SIMUFTA'
 3. input: 'First In,First Out'
      output: 'FIFO'

So any symbol except a-z A-Z and spaces should be treated as separator and be omitted from acronym

File: acronym/acronym_test.rb [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+    end
+  end
+
+
+aronym = Acronym.new
+ARGV = aronym.abbreviate
```
Comment (dzmitry-aleinik-itechart) [inline acronym/acronym_test.rb]: We should avoid reassigning the constants especially system ones

**Review by dzmitry-aleinik-itechart — COMMENTED** [review]
Please, let's try to keep pr's as clean as possible, without any unrelated to the task code, like (Arguments class)

File: acronym/acronym_test.rb (hunk 1 above)
Comment (Captain-Vorobey) [reply acronym/acronym_test.rb:12]: These characters (like &, %, $) are ignored and only the first letters of each word are displayed, like FIFO. You can check it yourself.

File: acronym/acronym_test.rb [hunk 3]
```diff
@@ -0,0 +1,14 @@
+class Acronym
+
+    def initialize(str)
+        @str = str
+    end
+
+    def abbreviate
+        @str.scan(/\w+/).join(" ").split.map(&:chr).join.upcase
```
Comment (dzmitry-aleinik-itechart) [inline acronym/acronym_test.rb:8]: Just for comparison, my solution was
` str.gsub(/[^\w\s]/, ' ').split.map { |word| word.capitalize.chr }.join`

**Review by dzmitry-aleinik-itechart — APPROVED** [review]

File: acronym/acronym_test.rb (hunk 3 above)
Comment (Captain-Vorobey) [reply acronym/acronym_test.rb:8]: Can I merge my solution to master now? Or you have to do it?

**Pull request merged**